src/components/DeleteBook.tsx

```diff
@@ -16,7 +16,7 @@ const DeleteBook: FC<IBookProps> = ({ id, onBookDelete, title }) => {
       .from('books')
       .delete()
       .match({ id });
-      setErr(error);
+    setErr(error);
```

Comment:
Is it necessary to store it in useState? You could just execute toast there
```suggestion
    error && errorToast(msg, id);
```